src/tactic/doc_commands.lean

```diff
@@ -120,6 +120,9 @@ attribute.get_instances `library_note >>=
 inductive doc_category
 | tactic | cmd | hole_cmd | attr
 
+instance : inhabited doc_category :=
```

Comment:
You can use `@[derive inhabited]` instead (it will produce exactly the same instance).

If you can't afford to `import tactic.derive_inhabited`, you can also just add it to the list there.

Reply:
Ah, cool. I've added both to the list in `derive_inhabited`.